### ISSUE TITLE
Add "Arisa" project for messages that are usually only used by Arisa.

### ISSUE THREAD

**Issue by MikeUnverferth** [mojira/helper-messages#96]
This will shorten the selection list for the other projects.

**Comment by NeunEinser** [comment]
@SPGoding pinging you since you implemented these things over on Arisa.
If I understand correctly, currently this is not possible, as Arisa always looks for the correct message depending on the project. I think it would be good to create a mirror of this issue on the Arisa project to highlight the changes that would be needed on Arisa's side.

**Comment by MikeUnverferth** [comment]
Alternatively instead of changing Arisa to look in another "project", we could have a way to simply exclude the Arisa-only messages from the selection list.

**Comment by SPGoding** [comment]
Yeah this is already a thing. For example, [the "I am a bot" message](https://github.com/mojira/helper-messages/blob/master/assets/messages.yml#L671) is hidden from the selection list. Do you think there are other messages that should be hidden?
